x-pack/metricbeat/metricbeat.reference.yml

```diff
@@ -427,6 +427,15 @@ metricbeat.modules:
   hosts: ["localhost:8500"]
 
 
+#------------------------------ Containerd Module ------------------------------
+- module: containerd
+  metricsets: ["cpu", "memory", "blkio"]
+  enabled: false
+  period: 10s
+  hosts: ["localhost:1338"]
```

Comment:
Is this default endpoint of getting containerD metrics? In docker the default is a socket. I think we lack documentation on how/fromWhere one collect containerD metrics.